routing/RouteSolver.py

```diff
@@ -0,0 +1,134 @@
+##################################################################################
+##                                                                              ##
+##   Program: Basic Route Solver for Factory Roaming Robot                      ##
+##   Author: Collin Duddy                                                       ##
+##   Class: EECS_542 Senior Design University of Kansas                         ##
+##                                                                              ##
+##   Description: This program will take in user specified input of a graph,    ##
+##                or map of nodes. It will then calculate all routes possible   ##
+##                from the users start node to the users end-node. This data    ##
+##                will be piped to a Raspberry Pi that will then send the       ##
+##                data to an arduino for roaming the map.                       ##
+##################################################################################
+import Graph
+import sys
+import re
+##################################################################################
+##                                                                              ##
+##                            Global Variables                                  ##
+##                                                                              ##
+##################################################################################
+graph = Graph.Graph()
+##################################################################################
+##                                                                              ##
+##                            Functions/Methods                                 ##
+##                                                                              ##
+##################################################################################
+
+def is_int(s):
```

Comment:
Get ride of this. Use builtin `isinstance`

Reply:
You can simply get rid of this. I explain below a better way of handling the issue you are trying to solve here